File: Semester 4/Programming and Projection Enironments/mpp-proiect-java/Proiect_MPP/AirportModel/src/main/java/root/model/DTODestinationDate.java
package root.model;

import java.io.Serializable;
import java.time.LocalDate;

public class DTODestinationDate implements Serializable {
    private final String destination;
    private final LocalDate date;

    public DTODestinationDate(String destination, LocalDate date) {
        this.destination = destination;
        this.date = date;
    }

    public String getDestination() {
        return destination;
    }

    public LocalDate getDate() {
        return date;
    }

    @Override
    public String toString() {
        return "DTODestinationDate{" +
                "destination='" + destination + '\'' +
                ", date=" + date +
                '}';
    }
}

File: Semester 4/Programming and Projection Enironments/mpp-proiect-csharp/Proiect_MPP/model/DTODestinationDate.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace model
{
    [Serializable]
    public class DTODestinationDate
    {
        public string Destination { get; }
        public DateTime Date { get; }

        public DTODestinationDate(string destination, DateTime date)
        {
            Destination = destination;
            Date = date;
        }

        public override string ToString()
        {
            return "DTODestinationDate{" +
                "destination='" + Destination + '\'' +
                ", date=" + Date.ToString() +
                '}';
        }
    }
}
